harmony/src/adapter.ts

```diff
@@ -0,0 +1,77 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { Execute } from '@chainlink/types'
+import { Harmony } from '@harmony-js/core'
+import { ethers } from 'ethers'
+import { formatBytes32String } from '@harmony-js/contract'
+import { getAddress } from '@harmony-js/crypto'
+import { ChainType, strip0x } from '@harmony-js/utils'
+
+const DEFAULT_ENDPOINT = 'https://api.s0.t.hmny.io'
+const hmyRPCEndpoint = process.env.HMY_NODE_URL || DEFAULT_ENDPOINT
+
+const DEFAULT_CHAIN_ID = 1
+const chainID = Number(process.env.HMY_CHAIN_ID) || DEFAULT_CHAIN_ID
+
+const DEFAULT_GAS_LIMIT = 6721900
+const gas = process.env.GAS_LIMIT || DEFAULT_GAS_LIMIT
+
+const hmy = new Harmony(hmyRPCEndpoint, {
+  chainType: ChainType.Harmony,
+  chainId: chainID,
+})
+const DUMMY_KEY = 'fd416cb87dcf8ed187e85545d7734a192fc8e976f5b540e9e21e896ec2bc25c3'
+const privateKey = process.env.PRIVATE_KEY || DUMMY_KEY
+hmy.wallet.addByPrivateKey(privateKey)
```

Comment:
Could you please move all of this configuration into a separate Config object, like this? https://github.com/smartcontractkit/external-adapters-js/blob/530a4552aeddc4abf4357e373d58d6d4ba6b7135/dydx-stark/src/config.ts

This will make it easier for us to give it different config params to test with, and makes it easier if we need to import this adapter in a different package.